digitalocean/baseapi.py

```diff
@@ -97,6 +97,32 @@ def __perform_request(self, url, type=GET, params=None):
 
         return requests_method(url, **kwargs)
 
+    def __deal_with_pagination(self, url, method, params, data):
+        """
+            Perform multiple calls in order to have a full list of elements
+            when the API are "paginated". (content list is divided in more
+            than one page)
+        """
```

Comment:
Why the consistency of the single quote comments is not kept here? Suggestion: revert the changes related to double-single quote, and just focus on the problem that has been fixed in this section of the code! It is hard to review!